lib/phoenix/live_dashboard/components/card_component.ex

```diff
@@ -15,7 +15,7 @@ defmodule Phoenix.LiveDashboard.CardComponent do
   defp validate_required(params, list) do
     case Enum.find(list, &(not Map.has_key?(params, &1))) do
       nil -> :ok
-      key -> raise ArgumentError, "#{inspect(key)} parameter expected"
+      key -> raise ArgumentError, "the #{inspect(key)} parameter is expected in card component"
```

Comment:
Note we will need to do similar to change to similar error messages below. :)

Reply:
> Note we will need to do similar to change to similar error messages below. :)

Done. Please check again.